src/main/resources/ath-container/Dockerfile

```diff
@@ -32,15 +32,16 @@ RUN apt-get update && \
     apt-get clean all && rm -rf /var/cache/apt
 
 # Install a fixed firefox version that is known to work with the current selenium version, copied from https://hub.docker.com/r/selenium/node-firefox/dockerfile
-ARG FIREFOX_VERSION=108.0.2
+ENV FIREFOX_VERSION 108.0.2
```

Comment:
nit: is this hadolint which proposed to switch from `ARG` to `ENV`?

I would have expected `ARG` to be used to avoid persisting the variable in the final image but I might have missed something here?

Reply:
I've restored `ARG`, hadolint doesn't complain locally anymore. Maybe I falsely suppresed it here.